docs/building/unix-instructions.md

```diff
@@ -56,3 +56,5 @@ $ brew update && brew install cmake https://raw.githubusercontent.com/dotnet/mac
 ```
 
 Please note that newer versions of Homebrew [don't allow installing directly from a URL](https://github.com/Homebrew/brew/issues/8791). If you run into this issue, you may need to download libomp.rb first and install it with the local file instead.
+
+Also, libomp version 7.0.0 doesn't have a cask for Big Sur. You can work around this by downloading the libomp.rb file and then calling `brew install libomp.rb --build-from-source --formula`.
```

Comment:
Presumably people will already have cloned the repo, so they can use that file rather than download it.   Maybe we could even move the install steps to a bash script that's checked in and just tell folks to invoke that?  Maybe make the CI process do the same?

Reply:
This step is only needed for Big Sur, the other versions of the OS work fine, Do we know the percentage of mac users on Big Sur compared to older versions? If its the majority I think that would be worth looking into.